MovieAppPetProject/MovieAppPetProject/CoreDataFiles/WatchList+CoreDataClass.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  WatchList+CoreDataClass.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/05/14.
```

Comment:
If these files are not needed manually, you can set it to not store in the project by ticking class definition instead of manual when creating them

Reply:
can I do tick it and remove them now knowing they'll be safe? since they're already there

Reply:
not sure what you're trying to say. Can you explain it again?

Reply:
sorry, what I mean is since the manual files are already there now, is safe / ok for me to remove the manual files and then just tick class implementation for the entity so that it's in the background like you said and it will be ok? or is there a specific way of removing them without breaking anything.

Reply:
yeah should be able to